src/app/content/highlights/components/cardUtils.ts

```diff
@@ -71,3 +71,12 @@ export const generateUpdatePayload = (
 
   return {updatePayload, preUpdateData};
 };
+
+export const noopKeyCombinationHandler = (activeElement: Element | null): boolean => {
+  if (!activeElement) { return false; }
+  if (activeElement.nodeName === 'TEXTAREA') { return true; }
+  if (activeElement.nodeName === 'INPUT') {
+    return activeElement.getAttribute('type') === 'text';
```

Comment:
what about other text like inputs such as 'password'

Reply:
My intent was to allow passing `noopHandler` as an argument to `useKeyCombination` so this function is passed only for a hook that is used specifically in the `CardWrapper` so we don't have to care about other cases
In Card components we have textareas and checkboxes

Reply:
but no text inputs right? so if its supposed to be super specific for that use case then why are you checking for text inputs?

Reply:
right, that can be confusing, we should either check for all text like inputs or check only for specific ones